### ISSUE TITLE
Change loss function

### ISSUE THREAD

**Issue by ZeroMaxinumXZ** [minimaxir/gpt-2-simple#112]
New to ML, so...

Is there any way to change the loss function of GPT2? I'm wanting to use an information-maximization loss function for the model, if possible...

**Comment by minimaxir** [comment]
tl;dr no. We can change the _optimizer_ (not implemented), but the loss function for this type of task should always be categorical crossentropy.
